src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/test_resource.py

```diff
@@ -67,63 +67,48 @@ def body(self):
         s.cmd('group exists -n {}'.format(rg), checks=NoneCheck())
 
 
-class ResourceScenarioTest(ResourceGroupVCRTestBase):
-    def test_resource_scenario(self):
-        self.execute()
-
-    def __init__(self, test_method):
-        super(ResourceScenarioTest, self).__init__(__file__, test_method,
-                                                   resource_group='cli_test_resource_scenario')
-        self.vnet_name = 'cli-test-vnet1'
-        self.subnet_name = 'cli-test-subnet1'
+class ResourceScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_rsrc_scenario')
```

Comment:
Create the resource group in "southcentralus" as well. And shorten the prefix. If the default prefix works, use the default one.

Reply:
errr, @tjprescott recommends prefixes. What to do?

Reply:
Travis always won.

Reply:
Just make sure the resource group is in southcentralus (a.k.a Texas) then.

Reply:
HAHAH okay.

Reply:
There's no harm in NOT using the prefix, but if you ever want to debug into a test as it records and want to inspect the resource group, pure gibberish makes it hard to find in the portal. The prefix makes it much easier. That's the main reason all my tests have one.